fhttp/http_test.go

```diff
@@ -733,13 +733,12 @@ func TestBadURLStdClient(t *testing.T) {
 }
 
 func TestDefaultPort(t *testing.T) {
-	// TODO: change back to fortio demo server once setup
-	url := "http://istio.io/" // shall imply port 80
+	url := "http://demo.fortio.org/" // shall imply port 80
 	opts := NewHTTPOptions(url)
 	cli, _ := NewFastClient(opts)
 	ctx := context.Background()
 	code, _, _ := cli.Fetch(ctx)
-	if code != 301 {
+	if code != 303 {
```

Comment:
Update next comment about expecting 301

Reply:
opsa yes, I switched back to demo.fortio.org and cloudflare does 303 not 301 for https redirect

Reply:
fixed now, thx